Fix some typos

NeodarZ opened the pull request:
Hi,

I just find a typo in the README file.

It's about the `refresh_cache_on_startup` option in the [Known issues](https://github.com/azlux/botamusique#known-issues) section.

If I see other before you merge it I will update this pull request.

TerryGeng commented:
Thanks for your help! Both @azlux and I are non-native English speakers and we are both busy somehow, so it's impossible to check all spelling.

I'll wait until you think you've done and merge this PR then.

NeodarZ commented:
No problem, I found the README well written.
 
I'm also a non-native English speaker so don't blindly accept what I fixed. 

I also fixed two more typos and I don't see more for the moment. You can merge it.
